server.js

```diff
@@ -301,20 +301,27 @@ app.get('/questionform', function(req, res) {
         return res.redirect('/');
     }
 
-    var html = questionForm();
-
-    return res.status(200).send(html);
-});
-
-app.get('/re', function(req, res){
-    var html = regexForm();
-    res.status(200).send(html);
+    return res.status(200).render('question-creation',{
+        questionType: common.questionTypes
+    });
 });
 
-app.get('/mc', function(req, res){
-    var html = multipleChoiceForm();
-    res.status(200).send(html);
-});
+app.post('/formanswer', function(req, res){
+    switch (req.body.qType){
+        case common.questionTypes.REGULAR.value:
+            res.status(200).render(
+                common.questionTypes.REGULAR.template,{
+                answerForm:true});
+            break;
+        case common.questionTypes.MULTIPLECHOICE.value:
+            res.status(200).render(
+                common.questionTypes.MULTIPLECHOICE.template,{
+                answerForm:true});
+            break;
+        default:
+            return res.redirect('/');
```

Comment:
hmmm this seems weird. What are our options here?

Reply:
This is how we send back the specific type of question that the user wants to see. Theres no better way of doing this. I have tried

Reply:
I think we should probably be sending this to some 404 page

Reply:
yeah, ahmed is working on that